class1/README.md

```diff
@@ -1 +1,131 @@
+
+##Javascript
+Many of you are likely familiar with Javascript, but don't worry if you aren't. Javascript, despite what the name suggests, has almost no connection to Java. It is the only actual programming language that runs within your browser itself. syntactically, it is probably one of the more *loose* programming languages and looks similar to python. It was originally designed for being used for quick, small, lightweight tasks within browsers and has since grown into its own language. Only in relatively recent years have personal computers and browsers gotten good enough to safely be able to run more serious Javascript code. It is a scripted language (meaning it executes line by line rather than compiling the whole program first) and has a relatively unique object-oriented implementation. You will get to hear A LOT more about Javascript in the coming weeks, so we will leave it here for now. 
+##Node
+Acording to the official website Node.js is:<br/>
+"Node.js® is a platform built on Chrome's JavaScript runtime for easily building fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient, perfect for data-intensive real-time applications that run across distributed devices." <br />
+What this really means is that it is Javascript running on the server, accepting incoming requests and handling them. The thing that makes Node.js so awesome, is that it is entirely **non-blocking**. To understand exactly what this means, realize that with other server frameworks, when a request comes in, the server has to process that request and send the response before it can accept the next incoming request. Node is structured in such a way that when a request comes in, it can immediately continue listening for other requests. 
+
+##Setup for homework
+We are going to be using git for homework submission. The way this will work is that you will fork this git repo into your own account and then you will push your homework to that repository when you finish it. You will next add another remote to your 
+
+### 1. Forking
```

Comment:
I'm going to steal this and put it into the class0 readme as well since ideally students will do this before class. This will let us include a question in the pre-class survey that asks the url of each student's repo (for easy access for us).
I think it make sense to leave it here as well so we can go over it in class if needed. Sound good?